packages/eas-cli/src/commands/channel/rollout.ts

```diff
@@ -117,7 +117,7 @@ export default class ChannelRollout extends EasCommand {
       required: false,
     }),
     'private-key-path': Flags.string({
-      description: `File containing the PEM-encoded private key corresponding to the certificate in expo-updates' configuration. Defaults to a file named "private-key.pem" in the certificate's directory.`,
+      description: `File containing the PEM-encoded private key corresponding to the certificate in expo-updates' configuration. Defaults to a file named "private-key.pem" in the certificate's directory.  Only relevant if you are using code-signing https://docs.expo.dev/eas-update/code-signing/`,
```

Comment:
```suggestion
      description: `File containing the PEM-encoded private key corresponding to the certificate in expo-updates' configuration. Defaults to a file named "private-key.pem" in the certificate's directory.  Only relevant if you are using code signing: https://docs.expo.dev/eas-update/code-signing/`,
```

(same with the others)